src/scripts/OSFramework/OSUI/Pattern/AbstractProviderPattern.ts

```diff
@@ -35,6 +37,13 @@ namespace OSFramework.OSUI.Patterns {
 			this.isProviderBased = true;
 		}
 
+		// Method that will check if there is already a redraw process runing in order to cancel it
+		private _cancelRedraw(): void {
+			if (this._refreshTimeOutId !== undefined) {
```

Comment:
we're not resetting the attribute `this._refreshTimeOutId` to `undefined`...

Reply:
🤦 Thank you!

Reply:
Done!